games.json

```diff
@@ -967,7 +967,9 @@
         "status": "Broken",
         "reference": "",
         "anticheats": [
-            "miHoYo Protect 2"
+            "miHoYo Protect 2 (Phasing Out)",
+            "miHoYo Protect 3",
+            "miHoYo Kernel Protect"
```

Comment:
Imo it's not necessary to go to deep into detail here, summarizing all of the AntiCheat Names into one would be more simple.

Reply:
Fair.